packages/core/src/actions/smartAccount/sendTransaction.ts

```diff
@@ -1,35 +1,30 @@
-import type {
-  Chain,
-  Client,
-  Hex,
-  SendTransactionParameters,
-  Transport,
-} from "viem";
+import type { Chain, Client, Hex, Transport } from "viem";
 import type { SmartContractAccount } from "../../account/smartContractAccount.js";
 import { isBaseSmartAccountClient } from "../../client/isSmartAccountClient.js";
 import { AccountNotFoundError } from "../../errors/account.js";
 import { IncompatibleClientError } from "../../errors/client.js";
 import { TransactionMissingToParamError } from "../../errors/transaction.js";
 import { buildUserOperationFromTx } from "./buildUserOperationFromTx.js";
 import { _sendUserOperation } from "./internal/sendUserOperation.js";
+import type { SendTransactionParameters } from "./types.js";
 import { waitForUserOperationTransaction } from "./waitForUserOperationTransacation.js";
 
 export const sendTransaction: <
+  TTransport extends Transport = Transport,
   TChain extends Chain | undefined = Chain | undefined,
   TAccount extends SmartContractAccount | undefined =
     | SmartContractAccount
-    | undefined,
-  TChainOverride extends Chain | undefined = Chain | undefined
+    | undefined
 >(
-  client: Client<Transport, TChain, TAccount>,
-  args: SendTransactionParameters<TChain, TAccount, TChainOverride>
+  client: Client<TTransport, TChain, TAccount>,
+  args: SendTransactionParameters<TAccount>
```

Comment:
does. this still build? this was the `viem` type before because it caused typing issues without it (because there was a conflict between the public client action of the same name